source/release-notes.txt

```diff
@@ -10,6 +10,41 @@ Release Notes
    :depth: 1
    :class: twocols
 
+1.3.4 Changelog
+---------------
+
+*Released December 11, 2023*
+
+Improvements:
+
+- Improved SQL query conversion quality for the Query Converter 
+  (private preview). For details on how to join the Query Converter 
+  early access program, see :ref:`rm-query-converter`.
+- Improved application performance through expanded use of available 
+  memory in suitable server environments.
+
+New features:
+
+- `Sybase ASE database 
+  <https://www.sap.com/products/acquired-brands/what-is-sybase.html>`__ 
+  support is now available for one-time snapshot sync jobs and 
+  stored procedure conversion.
+- `Confluent-hosted Kafka support <https://www.confluent.io/>`__ 
+  for participants in the Kafka early access program. This release 
+  adds prebuilt support for connecting to a Kafka cluster managed by 
+  the Confluent hosting provider. 
+  
+  .. note::
+
+    Kafka is an open-source platform for distributed, 
+    high-availability workloads. To gain access to Relation Migrator 
+    with Kafka support, contact your MongoDB account team to sign up 
```

Comment:
Should we add a link here to the MongoDB contact form? 

https://www.mongodb.com/contact